TCAT/NotificationToggleTableViewCell.swift

```diff
@@ -68,12 +77,24 @@ class NotificationToggleTableViewCell: UITableViewCell {
         }
     }
 
-    func configure(for notificationTitle: String, isFirst: Bool) {
+    func configure(for notificationTitle: String, isFirst: Bool, delegate: NotificationToggleTableViewDelegate? = nil) {
+        self.delegate = delegate
+        title = notificationTitle
         notificationTitleLabel.text = notificationTitle
         if isFirst {
             setupFirstHairline()
         }
     }
+    
+    @objc func switchValueChanged() {
+        if notificationSwitch.isOn {
+            if title == Constants.Notification.notifyBeforeBoarding {
```

Comment:
instead of passing in a `title` inside the `configure` and then comparing strings here, could we pass in a `type: NotificationType` and just add string associated values with each case in the enum? that way we would be comparing enums rather than strings which can sometimes be kinda risky